WeakAuras/WeakAuras.lua

```diff
@@ -3889,6 +3889,29 @@ function WeakAuras.GetAuraTooltipInfo(unit, index, filter)
   end
 end
 
+function WeakAuras.GetAuraTooltipInfo(unit, index, filter)
+  local tooltipText = ""
+  if WeakAuras.IsRetail() then
+    local tooltipData = C_TooltipInfo.GetUnitAura(unit, index, filter)
+    local secondLine = tooltipData and tooltipData.lines[2] -- This is the line we want
+    if secondLine then
+      for _, arg in ipairs(secondLine.args) do
+        if arg.field == "leftText" then
+          tooltipText = arg.stringVal or ""
+        end
+      end
+    end
```

Comment:
Is this retail block here only to not break custom auras that could use it WeakAuras.GetAuraTooltipInfo?

Reply:
Kinda, the check it's not really needed, as it shouldn't be possible to call the code on non-retail. So the check for retail can be removed.

Reply:
i'v just closed a ticket that use it on retail https://github.com/WeakAuras/WeakAuras2/issues/4314 so i'm ok not break custom code calling it for this wow extension